onnxruntime/core/providers/cpu/math/gemm.cc

```diff
@@ -8,7 +8,14 @@ namespace onnxruntime {
 ONNX_CPU_OPERATOR_VERSIONED_KERNEL(
     Gemm,
     7,
+    8,
+    KernelDefBuilder().TypeConstraint("T", DataTypeImpl::GetTensorType<float>()),
+    Gemm<float>);
+
+// input 'C' was made optional
```

Comment:
nit: 

wasn't input C made optional in opset 11 ? (comment for me to add in the other PR) 

I think the only diff from Gemm-7 to Gemm-9 is the extended type support...